tasks/src/audio-to-audio/data.ts

```diff
@@ -29,25 +29,25 @@ const taskData: TaskData = {
 	libraries: TASKS_MODEL_LIBRARIES["audio-to-audio"],
 	metrics:   [
 		{
-			description: "Signal-to-Noise ratio is the ratio of level of target signal to level of background noise.  It is calculated as the logarithm of target signal divided by background noise, in decibels.",
```

Comment:
the metric itself is called Signal-to-Noise Ratio so I'd keep the old one

Reply:
I liked the proposed changes, but indeed it should say ratio